internal/templates/login.html.tmpl

```diff
@@ -11,7 +11,13 @@
           <form id="login" action="/login?next={{.Next}}" method="POST">
             <fieldset>
               <label for="username">Username</label>
-              <input type="text" name="username" placeholder="Enter your username">
+              <input type="text" name="username" placeholder="Enter your username" value="{{ .Username }}">
+
+              <!-- "remember username" checkbox -->
+              <input type="checkbox" name="remember" id="remember">
```

Comment:
Minor preference to check that by default. The "home" use case is more likely one where you have a trusted device?

Reply:
Yep, makes sense IMO.

Reply:
Hm, actually, forcing the user to uncheck this every time if they don't want to remember seems annoying... I made it persistent, though, so if you previously clicked "remember", the next time you visit the login page it's pre-filled (and thus won't clear itself on next login).

Leaving a TODO for myself to think about this.

Reply:
Leaving it checked if it was ever checked seems good enough!